src/components/shared/Entity/LogDialogActions.tsx

```diff
@@ -13,35 +12,51 @@ interface Props {
     };
 }
 
+const buttonSX: SxProps<Theme> = {
+    mx: 1,
+};
+
 function LogDialogActions({ close, materialize }: Props) {
-    const entityCreateStore = getStore(useRouteStore());
+    const entityCreateStore = useRouteStore();
 
-    const formStatus = entityCreateStore(createStoreSelectors.formState.status);
+    const formStatus = entityCreateStore(
+        entityCreateStoreSelectors.formState.status
+    );
 
     console.log(formStatus);
 
     return (
         <>
-            <Status />
-
-            <Button
-                disabled={
-                    formStatus === FormStatus.TESTING ||
-                    formStatus === FormStatus.SAVING
-                }
-                onClick={close}
+            <Box
+                sx={{
+                    pl: 2,
+                }}
             >
-                <FormattedMessage id="cta.close" />
-            </Button>
+                <Status />
+            </Box>
 
-            {materialize ? (
+            <Box>
                 <Button
-                    disabled={formStatus !== FormStatus.SUCCESS}
-                    onClick={materialize.action}
+                    disabled={
+                        formStatus === FormStatus.TESTING ||
+                        formStatus === FormStatus.SAVING
+                    }
+                    onClick={close}
+                    sx={buttonSX}
                 >
-                    <FormattedMessage id={materialize.title} />
+                    <FormattedMessage id="cta.close" />
                 </Button>
-            ) : null}
+
+                {materialize ? (
+                    <Button
+                        disabled={formStatus !== FormStatus.SUCCESS}
+                        onClick={materialize.action}
+                        sx={buttonSX}
+                    >
+                        <FormattedMessage id={materialize.title} />
+                    </Button>
+                ) : null}
```

Comment:
Moved actions to common place. If there is the materialize function passed we show the button